app/src/routes/v1/keycloak.js

```diff
@@ -1,24 +1,20 @@
-const config = require('config');
-
+const keycloakRouter = require('express').Router();
 const Problem = require('api-problem');
 
 const keycloak = require('../../components/keycloak');
-const KeyCloakServiceClientManager = require('../../components/keyCloakServiceClientMgr');
-const RealmAdminService = require('../../components/realmAdminSvc');
-
-const keycloakRouter = require('express').Router();
+const clients = require('../../components/clients');
 
 // fetches all the service clients for all KC realms
 // current user must have role GETOK_ADMIN
 keycloakRouter.get('/serviceClients', keycloak.protect('realm:GETOK_ADMIN'), async (req, res) => {
 
-  const clients = await Promise.all([
-    getClientsFromEnv('dev'),
-    getClientsFromEnv('test'),
-    getClientsFromEnv('prod')
+  const serviceClients = await Promise.all([
+    clients.getClientsFromEnv('dev'),
+    clients.getClientsFromEnv('test'),
+    clients.getClientsFromEnv('prod')
   ]);
   // join them all into one array
-  const result = clients.flat();
+  const result = serviceClients.flat();
```

Comment:
You don't really need to create another temporary constant to flatten the array when it could be done in-line:
```js
  const result = await Promise.all([
    clients.getClientsFromEnv('dev'),
    clients.getClientsFromEnv('test'),
    clients.getClientsFromEnv('prod')
  ]).flat();
```

Reply:
yeah i tried that. but console tells me:
`TypeError: Promise.all(...).flat is not a function`

Reply:
Ah - might be order of operations. Let's leave it as is then.